linodecli/baked/request.py

```diff
@@ -9,7 +9,7 @@ class OpenAPIRequestArg:
     """
 
     def __init__(
-        self, name, schema, required, prefix=None, list_item=False
```

Comment:
I dropped `list_item` as an argument here since it's made redundant by checking whether `list_parent` is None.